src/status_im/contexts/communities/channel_details/view.cljs

```diff
@@ -0,0 +1,183 @@
+(ns status-im.contexts.communities.channel-details.view
+  (:require
+    [clojure.string :as string]
+    [quo.core :as quo]
+    [quo.foundations.colors :as colors]
+    [react-native.core :as rn]
+    [reagent.core :as reagent]
+    [status-im.common.contact-list-item.view :as contact-list-item]
+    [status-im.common.contact-list.view :as contact-list]
+    [status-im.common.home.actions.view :as actions]
+    [status-im.constants :as constants]
+    [status-im.contexts.chat.group-details.style :as style]
```

Comment:
I think I've raised this point a few times 🙃 
please don't re use styles like this. In React/ in general it is better to encapsulate styles in a component and reuse those components if you need shared stylings.

If it's not worth the compensation then it's better to just duplicate the styles to `status-im.contexts.communities.channel-details.style` otherwise we put a strange coupling on this component with some random styles